src/data/missions.csv

```diff
@@ -1153,3 +1153,28 @@
 "【デイリーミッション】\n育成を1回完了しよう","[Daily] Complete training mode once"
 "【デイリーミッション】\n育成でレースに勝利しよう","[Daily] Win a race in training mode"
 "【デイリーミッション】\nチーム競技場をプレイしよう","[Daily] Run in the team stadium once"
+"ログインしよう","Let's login"
+"イベントデイリーミッションを\nすべてクリアしよう","Complete all event daily missions"
+"育成でレースに\n5回勝利しよう","Win a race in training 5 times"
+"育成でレースに\n10回勝利しよう","Win a race in training 10 times"
+"育成でレースに\n20回勝利しよう","Win a race in training 20 times"
+"育成でレースに\n30回勝利しよう","Win a race in training 30 times"
+"育成でレースに\n40回勝利しよう","Win a race in training 40 times"
+"育成でレースに\n50回勝利しよう","Win a race in training 50 times"
+"育成でGⅠレースに\n3回勝利しよう","Win a GI race in training 3 times"
+"育成でファン数を\n累計10万人以上獲得しよう	","Earn a total of 100000 fans in training"
+"育成でファン数を\n累計50万人以上獲得しよう	","Earn a total of 500000 fans in training"
+"育成でファン数を\n累計100万人以上獲得しよう	","Earn a total of 1000000 fans in training"
+"育成でファン数を\n累計150万人以上獲得しよう","Earn a total of 1500000 fans in training"
+"育成で皐月賞の\n3着以内に入ろう","Finish 3rd or higher in Satsuki Sho during training"
+"育成で日本ダービーの\n3着以内に入ろう","Finish 3rd or higher in Japan Derby during training"
+"育成で菊花賞の\n3着以内に入ろう","Finish 3rd or higher in Kikka Sho during training"
+"育成で大阪杯の\n3着以内に入ろう","Finish 3rd or higher in Oosaka Hai during training"
+"育成で天皇賞（春）の\n3着以内に入ろう","Finish 3rd or higher in Tenno Sho (Spring) during training"
+"育成で宝塚記念の\n3着以内に入ろう","Finish 3rd or higher in Takarazuka Kinen during training"
+"育成で有馬記念の\n3着以内に入ろう","Finish 3rd or higher in Arima Kinen during training"
+"チーム競技場でスコアを\n累計20万以上獲得しよう","Earn a total of 200000 points in team stadium"
+"チーム競技場でスコアを\n累計50万以上獲得しよう","Earn a total of 500000 points in team stadium"
+"チーム競技場でスコアを\n累計100万以上獲得しよう","Earn a total of 1000000 points in team stadium"
+"チーム競技場でスコアを\n累計200万以上獲得しよう","Earn a total of 2000000 points in team stadium"
+"イベント期間限定ミッションを\nすべてクリアしよう","Complete all event limited missions"
```

Comment:
I don't see this added after importing from the new missions table, where did you get these?

Reply:
Okay I see it in category 190

Reply:
Let's keep it separately in `story-event-mission.csv`, I'll add it shortly